tasks/task497_esnli_classification.json

```diff
@@ -0,0 +1,444 @@
+{
+  "Contributors": ["Ian Lobo"],
+  "Source": [
+    "e-SNLI Dataset https://github.com/OanaMariaCamburu/e-SNLI/blob/master/dataset/esnli_test.csv"
+  ],
+  "Categories": ["Classification"],
+  "Definition": "Given Sentence 1 (the premise) and then given Sentence 2 (the hypothesis), determine if the hypothesis can be concluded from the premise and assigning it a label among entailment, neutral, or contradiction.",

```

Comment:
Please explain more about the entailment, neutral, and contradiction in the definition.